jargons/class.md

```diff
@@ -0,0 +1,20 @@
+---
+title: class
+tags:
+- c#
+- oop
+- object
+- instance
+---
+
+Nesne tabanlı ya da yönelimli programlama([oop](/oop)) yaklaşımında Sınıf'lar(class) kendisinden türetilecek Nesne'lerin([object](/object)) veri yapıları ve özelliklerini oluştururlar.
```

Comment:
parantezlerden once bir bosluk biraksak ve ozel kelime olmayan sinif ve nesneyi kesme isareti ile ayirmayip kucuk harflerle yazsak Turkce acisindan daha dogru olur mu acaba?

Reply:
Haklısın! Önerdiğin şekilde düzenliyorum. Ayrıca, parantez öncesi boşluk düzenlemesini daha önce eklediğim oop ve instance jargonları içinde yapıyorum.